src/prepack-standalone.js

```diff
@@ -36,10 +36,42 @@ Object.setPrototypeOf(InitializationError, Error);
 Object.setPrototypeOf(InitializationError.prototype, Error.prototype);
 Object.setPrototypeOf(FatalError.prototype, InitializationError.prototype);
 
+export function prepackSources(
+  sources: Array<SourceFile>, options: Options = defaultOptions
+): { code: string, map?: SourceMap, statistics?: SerializerStatistics } {
+  let realmOptions = getRealmOptions(options);
+  realmOptions.errorHandler = options.onError;
+  let realm = construct_realm(realmOptions);
+  initializeGlobals(realm);
+
+  if (options.serialize || !options.residual) {
+   let serializer = new Serializer(
+     realm,
+     getSerializerOptions(options),
+   );
+   let serialized = serializer.init(sources, options.sourceMaps);
+   if (!serialized) {
+     throw new FatalError();
+   }
+   if (!options.residual) return serialized;
+   let residualSources = [{ filePath: options.outputFilename || "unknown", fileContents: serialized.code,
+     sourceMapContents: JSON.stringify(serialized.map) }];
+   let result = realm.$GlobalEnv.executePartialEvaluator(residualSources, options);
+   if (result instanceof AbruptCompletion) throw result;
+   return (result: any);
```

Comment:
Sadly yes: Although statistics is an optional property, Flow complains about is absence from the result returned by executePartialEvaluator. That result is hard to fix since it is the result of Babel's generate function.